configs/smartling-config.json

```diff
@@ -4,10 +4,25 @@
             "application": "ar"
         },
                 
-    "locales": [{
+        {
             "smartling": "ms-MS",
             "application": "ms"
-        }
+        },
+                
```

Comment:
Not sure if this will cause problem, let's remove the extra space between each entry.